lib/Backend/Lower.cpp

```diff
@@ -2896,7 +2896,23 @@ Lowerer::LowerRange(IR::Instr *instrStart, IR::Instr *instrEnd, bool defaultDoFa
         case Js::OpCode::StatementBoundary:
             // This instruction is merely to help convey source info through the IR
             // and eventually generate the nativeOffset maps.
-            break;
+#if DBG_DUMP && DBG
+            // If we have a JITStatementBreakpoint, then we should break on this statement
+            {
+                uint32 statementIndex = instr->AsPragmaInstr()->m_statementIndex;
+                if (Js::Configuration::Global.flags.StatementDebugBreak.Contains(instr->m_func->GetFunctionNumber(), statementIndex))
```

Comment:
I've added the necessary machinery to have triples in a command line argument (in this case, sourceContextID, localFunctionID, statementIndex), and changed this check to use those values.